computers/daint-mc/daint-mc.yml

```diff
@@ -1,12 +1,13 @@
-label: "daint-mc"
-hostname: "daint.cscs.ch"
+---
+label: daint-mc
+hostname: daint.cscs.ch
 description: Piz Daint supercomputer at CSCS Lugano, Switzerland, multicore partition.
 transport: ssh
-scheduler: "slurm"
-work_dir: "/scratch/snx3000/{username}/aiida_run/"
-shebang: "#!/bin/bash"
-mpirun_command: "srun -n {tot_num_mpiprocs}"
-mpiprocs_per_machine: "36"
+scheduler: slurm
+work_dir: /scratch/snx3000/{username}/aiida_run/
+shebang: '#!/bin/bash'
+mpirun_command: srun -n {tot_num_mpiprocs}
+mpiprocs_per_machine: '36'
```

Comment:
I guess these quotation marks are introduced because we had quotation marks before, and it wants to keep the information that this should be treated as a string.
However, it would be perfectly fine to treat it as a number as well, i.e. these quotation marks (and those of all other numbers) can be removed.

Reply:
> I guess these quotation marks are introduced because we had quotation marks before

That true. But we can not depend on pre-commit to fix or check this, or we need to customize this rule ourselves (include other customize format errors such as checking invalid input keys).

Reply:
Ah, don't worry about this for new additions - I just wanted to make sure the examples we have in the repository are as clear as possible (i.e. just change it for the existing ones).
If people go by those examples, it will continue to be like this.